api/desecapi/tests/base.py

```diff
@@ -98,6 +100,28 @@ def delete_rr_set(self, domain_name, subname, type_):
     # TODO add and use {post,get,delete,...}_domain
 
 
+class ReadUncommitted:
+
+    def __init__(self):
+        self.read_uncommitted = None
+
+    def __enter__(self):
+        with connection.cursor() as cursor:
+            cursor.execute('PRAGMA read_uncommitted;')  # FIXME this is probably sqlite only?
```

Comment:
I guess it won't work with MySQL/MariaDB as those only support cursors inside stored procedures. Postgres supports cursors; I'm not sure though if the syntax is the same as for sqlite.

Reply:
This is definitely incompatible with MySQL and needs fixing. I'll address this later though, as I am running the tests currently just locally.